staticaddr/address/manager.go

```diff
@@ -74,7 +75,7 @@ func (m *Manager) Run(ctx context.Context) error {
 	for {
 		select {
 		case currentHeight := <-newBlockChan:
-			m.currentHeight = currentHeight
+			m.currentHeight.Store(currentHeight)
```

Comment:
I think we should also initialize the current height by querying best height at the start on `Run`.

Reply:
`Run` already does this as soon as possible. Actually this is the only thing Run does - keeping `currentHeight` up-to-date.